app/views/short_problems/_manage.html.haml

```diff
@@ -44,6 +50,18 @@
       = f.label :answer, class: 'col-sm-2'
       .col-sm-4
         = f.text_field :answer, class: 'form-control', autocomplete: 'off'
+    .form-group
```

Comment:
Can you make these input fields default to 1, 0, 0 respectively?

Same goes with the form on short_problems#edit.

Reply:
In my local machine, it is already there
![image](https://user-images.githubusercontent.com/56295230/108595274-8bfdbe00-73b1-11eb-8870-203bfb3f2f46.png)

Reply:
I see. That's probably from the database defaults, so I guess it should be all good